build/build.sh

```diff
@@ -284,10 +284,14 @@ if [[ $opt == 1 ]]; then
   fi
 
   if [[ $docs == 1 ]]; then
+    # Source the XRT environment so we can find modules like pyxrt
+    if [[ -z "${XILINX_XRT}" ]]; then
+      echo ERROR: Please source a XRT setup file before building the documentation
+    else
       echo "make xrt_docs"
-      # Source the XRT environment so we can find modules like pyxrt
-      source ./opt/xilinx/xrt/setup.sh
+      source ${XILINX_XRT}/setup.sh
       make xrt_docs
+    fi
```

Comment:
This doesn't look right.  Why this requirement?   While I understand the need for sourcing setup.sh to locate pyxrt, pointing to an external XILINX_XRT will not reflect pyxrt doc changes in current clone from where build.sh is run.    Maybe I missing something?

Reply:
It is probably me that is missing something. When I ran the original code I got a failure while generating the documentation stating that ./opt/xilinx/xrt/setup.sh did not exist.
In there I found two problems:
1. The '.' was not correct and the path should have been /opt/xilinx/xrt/setup.sh, in fact this would not even run!
2. The original source pointed to an the global installation directory. I agree that we should be pointing to the local directory. Maybe instead of ${XILINX_XRT} I should reference the release build using a relative path?

Reply:
Hi @dbenusov-xilinx .  No, the `.` is correct.  It is relative to the build directory.

Reply:
Aha got it you must have the release build completed, the debug build wont cut it. If it is alright with you I'll add a warning here stating to run the release build?

Reply:
Why will debug build not work?

Reply:
I see.  -docs is only supported in optimized.  This makes sense since it is the release build we want to document.